File: backups/widget_backup_from_build_failure/layout/RowElement.java

package io.github.abc15018045126.sora.widget.layout;

import io.github.abc15018045126.sora.lang.styling.inlayHint.InlayHint;

/**
 * Element on a row
 *
 * @author abc15018045126
 */
public class RowElement {

    /**
     * Type of element.
     *
     * @see RowElementTypes
     */
    public int type;

    /* Fields for type TEXT */

    /**
     * Start column of text
     */
    public int startColumn;
    /**
     * End column of text
     */
    public int endColumn;
    /**
     * Direction of the text run
     */
    public boolean isRtlText;

    /* Fields for type INLAY_HINT */

    /**
     * The inlay hint
     */
    public InlayHint inlayHint;

    /**
     * The expected column position to display after
     */
    public int displayColumnPosition;

}

File: android/sora-editor/editor/src/main/java/io/github/abc15018045126/sora/widget/layout/RowElement.kt
package io.github.abc15018045126.sora.widget.layout

import io.github.abc15018045126.sora.lang.styling.inlayHint.InlayHint


class RowElement {

    @JvmField
    var type: Int = 0




    @JvmField
    var startColumn: Int = 0


    @JvmField
    var endColumn: Int = 0


    @JvmField
    var isRtlText: Boolean = false




    @JvmField
    var inlayHint: InlayHint? = null


    @JvmField
    var displayColumnPosition: Int = 0
}
